amgut/db/ag.dbs

```diff
@@ -717,17 +720,21 @@
 			</fk>
 		</table>
 		<table name="zipcodes" >
-			<column name="zipcode" type="varchar" jt="12" mandatory="y" />
-			<column name="state" type="varchar" jt="12" />
-			<column name="fips_regions" type="varchar" jt="12" />
-			<column name="city" type="varchar" jt="12" />
-			<column name="latitude" type="float8" jt="6" />
-			<column name="longitude" type="float8" jt="6" />
-			<column name="elevation" type="float8" jt="6" />
-			<column name="cannot_geocode" type="bool" jt="-7" />
+			<column name="zipcode" type="varchar" length="5" jt="12" mandatory="y" />
```

Comment:
this should not be length 5. I have a feeling that this PR is reverting changes I made to these tables in my previous PR